cmd/create/cluster/cmd.go

```diff
@@ -807,6 +803,15 @@ func networkTypeCompletion(cmd *cobra.Command, args []string, toComplete string)
 }
 
 func run(cmd *cobra.Command, _ []string) {
+	flags := cmd.Flags()
+	flags.SortFlags = false
+	args := &args{}
+	args.bind(flags)
+
+	cmd.RegisterFlagCompletionFunc("network-type", networkTypeCompletion)
+	aws.AddModeFlag(cmd)
```

Comment:
note: fan-out of the changes proposed in this PR would also end up changing the other libraries, but that can come in individual PRs as follow-ups to keep the scope small here.